src/coffee.js

```diff
@@ -17,14 +17,15 @@ const coffee = [
     acidity: "round",
     url: "https://impactroasters.dk/pages/about-coffe-coffee-guide",
     image: AynuImage,
-    characteristics: ["fruit acidity", "nut aroma"],
+    characteristics: ["rounded", "fruit acidity", "nut aroma"],
```

Comment:
For the `characteristics` in general, this should be done:

Let's merge `citrus aroma` and `fresh citrus`, keep only `citrus`

Let's merge `fruit acidity` and `fruity`, keep only `fruity`

Let's get rid of `sweet` (it's also in acidity)

Let's get rid of `slightly acidic` (it's also in acidity)

Let's simplify these characteristic names:
- `nut aroma` -> `nut`
- `forest berry aroma` -> `forest berry`
- `nougat aroma` -> `nougat`
- `chocolate aroma` -> `chocolate`

So at the end of the day, the list of characteristics should looks something like this:
- `chocolate`
- `citrus`
- `delicate sweetness`
- `floral`
- `forest berry`
- `fruity`
- `honey`
- `intense sweetness`
- `nougat`
- `nut`
- `spicy`